examples/gallery/ui/pages/controls_page.slint

```diff
@@ -206,6 +206,10 @@ export component ControlsPage inherits Page {
                 VerticalBox {
                     Text {
                         text: @tr("Content of tab 3");
+                        font-size: 48px;
+                        stroke: #ff0000;
+                        stroke-width: 2px;
+                        stroke-style: i-progress-indicator.indeterminate ? outside : center;
```

Comment:
Sort of. I put this here initially to make it easy to test while I wrote the PR with the intention of reverting it. However, because there wasn’t any content on this tab beforehand, I left it in so it could be demoed somewhere in the gallery. With that said, I’m totally fine with reverting this if you’d like.

Reply:
Yes, I'd prefer this to be left out.